### PR TITLE
Feature styles

### PR DESCRIPTION
**Pull request by franpossetto**
Global styles have been added.

styles forlder contains all the files related with global css.
-voyansi.scss file links all the file that manage styling. You should not write your styles on this file. it only have to contain imports.
-variables/colors.scss file contains variables related with colors. You should use the the derivated variables. It is an easy way to understand where the color should be used.
-variables/colors.scss file contains voyansi custom classes for text.
-components/helpers.scss contains classes related with Flexbox.
-components/buttons.scss contains classes related with buttons. Create a scss file for a component if you think it could have many classes related.
-components/generic.scss contains all the code that are not in the described files.
-variables.scss use this file to modify vuetify sass variables.
-add settings page.

### CODE REVIEW

**Review by mpothier — CHANGES_REQUESTED** [review]
Nice work! This is a great styling infrastructure to use moving forward.
In addition to the in-line comments, here are a few requests/comments:
- The font weight is a bit too light in most cases. See if you can use multiple font weights for things like title/subtitle/body, etc.
- Change the sidebar color to something from the branding colors
- I think the purple primary color on the buttons feels a bit too dark. Is there a lighter shade from our branding colors, or another lighter accent color instead?
- Hold off on developing content for the Settings page for now
- Try updating your branch by merging in the latest commits from `dev`, which will hopefully resolve some of the errors you're seeing. Resolve any git conflicts you encounter (it looks like there will be one on the vuetify plugins file).

File: src/components/browse/ControlBar.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
     </v-col>
     <v-spacer />
     <v-col>
-      <v-btn-toggle mandatory v-model="tempDisplayToggle" id="toggle-buttons">
+      <v-btn-toggle mandatory v-model="tempDisplayToggle" class="vo-toggle-buttons">
```
[the remark this answers is not in the document]
Comment (franpossetto) [reply src/components/browse/ControlBar.vue:18]: Ok, I converted it into a class.

File: src/views/Settings.vue [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
+  location = "US";
+}
 </script>
 <style lang="scss">
 #container {
   background-color: rgb(224, 224, 224);
-  height: 100vh;
+  width: 50%;
 }
 </style>
```
[the remark this answers is not in the document]
Comment (franpossetto) [reply src/views/Settings.vue:63]: ok. perfect

File: src/styles/variables/fonts.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+.voyansi-font-subtitle {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.voyansi-font-text {
+  font-size: 10px;
+  font-weight: 500;
+}
```
[the remark this answers is not in the document]
Comment (franpossetto) [reply src/styles/variables/fonts.scss:14]: I don´t think so. I think it is a way to define our design, having some rules. For example, font-weight could be solved by assigning a weight to certain elements as Title, subtitle, text. 

Other alternative could be has classes with just one responsibility, for example:

.voyansi-weight-700 {
  font-weight: 700;
}

.voyansi-weight-600 {
  font-weight: 600;
}

.voyansi-weight-text {
  font-weight: 500;
}

What do you think?

File: src/styles/variables.scss [hunk 4]
```diff
@@ -1,14 +1,14 @@
 // Modify Vuetify variables in this file
 // List of sass variables https://vuetifyjs.com/en/api/vuetify/#sass-variables
 
-@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
 
 $body-font-family: "Poppins", serif;
-
-$btn-letter-spacing: 0;
+$btn-font-weight: 300;
```
Comment (mpothier) [inline src/styles/variables.scss:7]: Let's make the button text font weight a bit heavier (500?) to match the baseline text

**Review by mpothier — CHANGES_REQUESTED** [review]
A few more quick changes:
- change sidebar color to something from our branding colors
- change button color to a lighter version of the purple (assuming there's one from branding)
- change button text font weight to be slightly heavier
- remove package-lock.json file; if you added any new packages, make sure to use `yarn add ...` instead of `npm`

**Pull request merged**